docs/calculations.mdx

```diff
@@ -437,42 +437,55 @@ creative_consumer_device_embodied_emissions_gco2pm =
 ### Media Delivery - Data Transfer
 
 ```
-session_seconds_per_imp =
-      property.average_seconds_per_session_ex_ads x
-      property.ad_funded_percentage /
-      property.average_imps_per_session
-
 media_data_transfer_emissions_gco2pm =
       media_data_transfer_usage_emissions_gco2pm +
       media_data_transfer_embodied_emissions_gco2pm
+```
+
+Determing the session seconds for a single impressions
 
 ```
+session_seconds_per_imp =
+      property.average_seconds_per_session_ex_ads x
+      property.ad_funded_percentage /
+      property.average_imps_per_session
```

Comment:
@bokelley I think we need to add defaults for these values if we do not have them at the property level, likely broken out by channel

Reply:
We have decent session time data from similarweb/data.ai so we can pull an 80th percentile number and contribute. Feels like 100% is the right ad funded default (to be conservative), and we can pull same imps per session numbers that we had before (I think this is really a "imps per session second" default like we have now?)